RecoLocalCalo/Configuration/python/hgcalLocalReco_cff.py

```diff
@@ -10,10 +10,11 @@
 from RecoLocalCalo.HGCalRecProducers.hgcalMultiClusters_cfi import *
 from RecoLocalCalo.HGCalRecProducers.hgcalLayerClusters_cff import hgcalLayerClusters
 
-hgcalLocalRecoSequence = cms.Sequence( HGCalUncalibRecHit+
-                                       HGCalRecHit+
-                                       hgcalLayerClusters+
-                                       hgcalMultiClusters+
-                                       particleFlowRecHitHGCSeq+
-                                       particleFlowClusterHGCal+
+hgcalLocalRecoTask = cms.Task( HGCalUncalibRecHit,
+                                       HGCalRecHit,
+                                       hgcalLayerClusters,
+                                       hgcalMultiClusters,
+                                       particleFlowRecHitHGCTask,
```

Comment:
I wonder whether one couldn't simply add here ```particleFlowRecHitHGC``` from ```RecoParticleFlow.PFClusterProducer.particleFlowRecHitHGC_cfi```, instead of the task which only contains that module.

@lgray @felicepantaleo @rovere : was there a reason for having originally a sequence defined with one single module in ```RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHGC_cff.py```? (As a corollary: could that cff file be removed now when moving to unscheduled execution?)

Reply:
I cannot comment for the past history, but I see not special reason.
As for the Task, we could bypass it but I'm not sure if there is any added value.